scripts/components/LviEmptyItem.ts

```diff
@@ -0,0 +1,31 @@
+import { getCombinedStyle } from '@smartface/extension-utils/lib/getCombinedStyle';
+import Screen from '@smartface/native/device/screen';
+import LviEmptyItemDesign from 'generated/my-components/LviEmptyItem';
+
+export default class LviEmptyItem extends LviEmptyItemDesign {
+    pageName?: string | undefined;
+    constructor(props?: any, pageName?: string) {
+        // Initalizes super class for this scope
+        super(props);
+        this.pageName = pageName;
+    }
+
+    static getHeight(): number {
+        return getCombinedStyle(`.lviEmptyItem`).height || 0;
+    }
+    static getScreenHeight(): number {
+        return Screen.height / 2;
```

Comment:
definition and function doesn't make sense

Reply:
done.